tests/test_layout.py

```diff
@@ -1,4 +1,64 @@
 import pytest
+from alfasim_sdk.layout import tabs, tab
+from alfasim_sdk.types import String
+
+
+def test_group():
+    from alfasim_sdk.models import data_model
+    from alfasim_sdk.layout import group
+    @data_model(caption='Foo')
+    class ValidClass:
+        string_from_main = String(caption='Caption from Root', value='Root')
+
+
+        @group(caption='This is a Group')
+        class GroupMain:
+            string_from_group = String(caption='Caption From Group', value='Inside Group')
+
+
+    import attr
+    # Checking the Model has all attributes
+    assert len(attr.fields(ValidClass)) == 2
```

Comment:
You can replace all those 3 lines with:

```python
assert [x.name for x in attr.fields(ValidClass)] == ['string_from_main', 'GroupMain']
```

This also provides a better diagnosis when it fails. 😉